letsencrypt-nginx/letsencrypt_nginx/configurator.py

```diff
@@ -572,15 +573,24 @@ def perform(self, achalls):
         outstanding challenges will have to be designed better.
 
         """
-        self._chall_out += len(achalls)
-        responses = [None] * len(achalls)
         chall_doer = tls_sni_01.NginxTlsSni01(self)
 
         for i, achall in enumerate(achalls):
             # Currently also have chall_doer hold associated index of the
             # challenge. This helps to put all of the responses back together
             # when they are all complete.
-            chall_doer.add_chall(achall, i)
+            # As we does not create vhost when it isn't found in the current configuration,
+            # we will filter out this domains before to perform TLS challenge
+            if self.choose_vhost(achall.domain) is not None:
+                chall_doer.add_chall(achall, i)
+            else:
+              # TODO: Handle feedback about domain without vhost
+                logger.info("As was not found a virtual host for domain %s, "
```

Comment:
Can we change the string here to: "As we couldn't find a virtual host for domain %s, it will be ignored"